apis/v1alpha1/httproute_types.go

```diff
@@ -58,6 +58,16 @@ type HTTPRouteHost struct {
 	// +optional
 	Hostnames []string `json:"hostnames,omitempty"`
 
+	// TLS defines the TLS certificate to use for Hostnames defined in this
+	// Route. The certificate defined here takes effect only if the
```

Comment:
Maybe a small change since the certificate itself isn't actually defined here.
```suggestion
	// Route. This configuration only takes effect if the
```